### DISCUSSION TITLE
Handing down passwords in case of death or else

### DISCUSSION THREAD

**Discussion by jaker-dotcom** [keepassxreboot/keepassxc#6707] [Q&A] [answered]
Hey there! 
I wonder how I could set up for (the pessimistic) eventuality that I cease to exist or am suddenly unable to communicate due to illness or accident. 

I bet that some of the KeePassXC users have thought about this and perhaps someone came up with a strategy.
Providing someone with the database files is no problem as file sync is in place. The problem is how to grant others access only when I can't take no action no more? I mean I trust them enough but you never know how they would treat the password or keyfile (read oftentimes you know how badly). They could always have the database but only and for sure get the keyfile / passphrase at that time when I can't give it to them anymore.

Thanks in advance, hope I didn't disgruntle noone with this pessimistic scenario.
Jaker

**Comment by droidmonkey** [comment, marked as answer]
The choice is yours. Print it out on paper and keep in safety deposit box or safe. Use a dead man switch service. Etc.

**Reply by jaker-dotcom** [reply]
Good keyword I found lots of possible solutions. Thanks!

**Comment by alejandracios** [comment]
Let me resume this discussion. 
Is there a way to implement the features available on keepass 2 where you can print a page to store the password and also to recreate the keyfile? 
I would like to secure the database with a keyfile but in case something happens to me I would like my wife to open the database to close social media. If there is a page with instructions secured in my house would help to follow steps and open the database. 
Thanks!

**Reply by droidmonkey** [reply]
The keyfile, if created by KeePassXC, is just an XML file. Open it in a text reader and print.
